### ISSUE TITLE
Zero animation time for `SwitchRound` causes `ZeroDivisionError`

### ISSUE THREAD

**Issue by The-Compiler** [adafruit/Adafruit_CircuitPython_DisplayIO_Layout#75]
The docs [claim](https://docs.circuitpython.org/projects/displayio-layout/en/latest/api.html#feature-summary) that it's possible to set `animation_time` to zero:

> **animation:** Set `animation_time` to the duration (in seconds) it will take to transition the switch, set zero if you want it to snap into position immediately

However, the code doesn't actually handle that case, thus leading to a `ZeroDivisionError` here:

https://github.com/adafruit/Adafruit_CircuitPython_DisplayIO_Layout/blob/aa66d0fdd6b02e91b8e138dbfc12a7b3d04d3af3/adafruit_displayio_layout/widgets/switch_round.py#L767

Traceback:

```pytb
Traceback (most recent call last):
  [...]
  File "adafruit_displayio_layout/widgets/switch_round.py", line 830, in value
  File "adafruit_displayio_layout/widgets/switch_round.py", line 792, in selected
  File "adafruit_displayio_layout/widgets/switch_round.py", line 767, in _animate_switch
```

**Comment by tekktrik** [comment]
Adding this a good first issue so maybe someone will pick it up for Hacktoberfest.  The fix is to make sure that if the animation time is zero that the animation process is skipped.  This should look something like a check to see if `_animation_time` is zero, and if it is, calculate the final position, and avoid dividing by the animation time entirely.

I'll pick this up soon-ish if no one does soon!

**Comment by The-Debarghya** [comment]
Can you specify which files are involved?
I can work on this then.

**Comment by The-Compiler** [comment]
@The-Debarghya See the file I linked in my issue description - that `_animate_switch` method should be the only place needing a change. Often, changes will also require corresponding doc or tests changes. However, in this case, the docs are already correct, and there seem to be no existing tests (so I'm assuming you won't be expected to add a new one for this change).

**Comment by The-Debarghya** [comment]
Ok thanks, I'm trying to fix this.

**Comment by tekktrik** [comment]
Fixed via #76
